flexmeasures/data/schemas/account.py

```diff
@@ -0,0 +1,22 @@
+from flask.cli import with_appcontext
+from marshmallow import fields
+
+from flexmeasures.data.models.user import Account
+from flexmeasures.data.schemas.utils import FMValidationError, MarshmallowClickMixin
+
+
+class AccountIdField(fields.Int, MarshmallowClickMixin):
+    """Field that de-serializes to a Sensor and serializes back to an integer."""
+
+    @with_appcontext
+    def _deserialize(self, value, attr, obj, **kwargs) -> Account:
+        """Turn a source id into a DataSource."""
+        account = Account.query.get(value)
+        account.account_roles  # lazy loading now
```

Comment:
Why?

Reply:
This might actually be an interesting topic: If I don't do this, then the 'Account' instance which got deserialized is not in my session:

```
sqlalchemy.orm.exc.DetachedInstanceError: Parent instance <Account at 0x7f00db5e3100> is not bound to a Session; lazy load operation of attribute 'account_roles' cannot proceed (Background on this error at: https://sqlalche.me/e/14/bhk3)
```
I tried to change `Account.query.get(value)` to `db.session.query(Account).get(value)`, but that isn't the problem.

There is a chance that this is the same problem which leads to the CLI tests failing.

Maybe a marshmallow intricacy ...

Reply:
Both the CLI command and the deserialize function are using the same session (I checked), so somewhere the loaded object is expunged from the session, or something to that effect. I have no idea where, though ...

Reply:
I spent another 30 minutes on this with no effect. I'll add a comment.